samples/ControlCatalog/MainView.xaml.cs

```diff
@@ -50,6 +50,7 @@ public MainView()
                         }
                         Application.Current.Styles[0] = App.Fluent;
                         Application.Current.Styles[1] = App.DataGridFluent;
+                        Application.Current.Styles.Add(App.ColorPickerFluent);
```

Comment:
Should it be ".Add" call instead of indexer setter? Seems like it will only add same styles again and again on theme switching.

Reply:
No idea, not sure how this code really works and was going to ask once I got further along getting things to build. Where is length set though? I am hesitant to just use `[2]` because I don't know that index will actually exist.

Reply:
Fixed.